packages/editor-ui/src/components/Card.vue

```diff
@@ -1,20 +1,19 @@
 <template>
-	<div
+	<n8n-card
```

Comment:
Maybe we should rename this component to TemplateCard or WorkflowCard or something

Reply:
Renamed to WorkflowCard. (TemplateCard already existed)